src/winforms/toga_winforms/factory.py

```diff
@@ -17,9 +17,10 @@
 from .widgets.scrollcontainer import *
 from .widgets.selection import Selection
 from .widgets.slider import *
```

Comment:
At column 1: (F403) 'from .widgets.slider import *' used; unable to detect undefined names